src/main/java/org/broadinstitute/hellbender/utils/pairhmm/EstimateDragstrModelParameters.java

```diff
@@ -0,0 +1,158 @@
+package org.broadinstitute.hellbender.utils.pairhmm;
+
+import htsjdk.samtools.*;
+import htsjdk.samtools.util.BufferedLineReader;
+import htsjdk.samtools.util.LineReader;
+import org.apache.commons.collections4.EnumerationUtils;
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.barclay.argparser.ArgumentCollection;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.engine.GATKTool;
+import org.broadinstitute.hellbender.exceptions.GATKException;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.utils.gcs.BucketUtils;
+
+import static org.broadinstitute.hellbender.utils.pairhmm.DragstrConstants.*;
+
+import picard.cmdline.programgroups.ReferenceProgramGroup;
+
+import java.io.*;
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipFile;
+
+@CommandLineProgramProperties(
+        programGroup = ReferenceProgramGroup.class,
+        summary = "Determine the presence of STR in a reference sequence",
+        oneLineSummary = "Determines the presence of STR in a reference sequence"
+)
+public class EstimateDragstrModelParameters extends GATKTool {
```

Comment:
Complicated methods like this should have javadocs. Also add `@DocumentedFeature`